src/sentry/src/Aspect/SentrySdkAspect.php

```
@@ -0,0 +1,70 @@
+    public function process(ProceedingJoinPoint $proceedingJoinPoint)
+    {
+        // Do nothing, just for class mapping.
+        return match ($proceedingJoinPoint->methodName) {
+            'init' => $this->handleInit($proceedingJoinPoint),
+            'setCurrentHub' => $this->handleSetCurrentHub($proceedingJoinPoint),
+            'getRuntimeContextManager' => $this->handleGetRuntimeContextManager($proceedingJoinPoint),
+            default => $proceedingJoinPoint->process(),
+        };
+    }
+
+    private function handleInit(ProceedingJoinPoint $proceedingJoinPoint)
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$proceedingJoinPoint'. (undefined)

(UnusedFormalParameter)